iree/jax/jax_utils.py

```diff
@@ -28,7 +28,7 @@
 # Need to interop with the JAX version of MLIR, which may or may not be
 # what we have here.
 from jax._src.lib.mlir import ir as jax_ir
-from jax.interpreters.xla import abstractify as jax_abstractify
+from jax._src.api_util import shaped_abstractify
```

Comment:
From what I can tell this is functionally equivalent, but I haven't tested this outside of the tests that are included in the project.

https://github.com/google/jax/blob/main/jax/_src/api_util.py#L560